schemas/skills/samples/complex-skillmanifest-2.1.preview-1.json

```diff
@@ -106,6 +106,18 @@
         "message": {
             "type": "message",
             "description": "Receives the user's' utterance and attempts to resolve it using the skill's LU models"
+        },
+        "typing": {
+            "type": "typing"
+        },
+        "conversationUpdate": {
+            "type": "conversationUpdate",
+            "name": "memberAdded",
```

Comment:
I think we need to remove this, conversationUpdate doesn't have a name property set (it is null by default)